src/net/sourceforge/kolmafia/swingui/MuseumFrame.java

```diff
@@ -251,6 +251,7 @@ public void updateDisplay(final PanelList list, final Object value, final int in
   }
 
   public class OrderingPanel extends ItemListManagePanel<String> {
+    @SuppressWarnings("unchecked")
```

Comment:
Why are you adding this

Reply:
See the PR's comment:

> For LockableListModel... ugh... just ignore the warning, I guess...

This clone is a LockableListModel, which is one of the files we have in /lib
Its `clone` method still returns an `Object`, so we can't have the type-safety be asserted earlier.

Reply:
Please add a short but clear comment on the previous (or next - does Java allow that?) line explaining why you've added it. This is general advice for all `SuppressWarnings` annotations.